test/e2e/tests/metamask-responsive-ui.spec.js

```diff
@@ -104,9 +104,10 @@ describe('MetaMask Responsive UI', function () {
         await driver.press('#confirm-password', driver.Key.ENTER);
 
         // balance renders
+        const balance = await ganacheServer.getBalance();
```

Comment:
here we access the current balance of the acocunt, instead of hardcoding the balance